SAP/open-ux-tools
# feat(fe-fpm-writer): support to generate table building block

# Description

* Add support to generate [table building block](https://ui5.sap.com/#/api/sap.fe.macros.Table%23overview)
* Move existing `basic-lrop` integration test app to a nested folder. 
> Note: the `basic-lrop-app` integration test app seems to be broken while previewing in both `js` and `ts` variants
* Add new `custom-page-app` integration test app to test filter bar, chart and table building blocks

## Steps to preview new integration test app
* Run tests in `fe-fpm-package` using the debug environment variable - `UX_DEBUG=true pnpm test`
* Change directory into `test/test-output/integration/custom-page-app/js` or `test/test-output/integration/custom-page-app/ts`
* Run `npm install` followed by `npm run start-mock`
* Hit the Go button
* The application should contain a filter bar, an empty chart (due to missing mock data) and a table with mock data
